docs/usage/modules/manager/index.md

```diff
@@ -18,38 +18,42 @@ The goal of Renovate is to detect and maintain all third-party dependencies in y
 
 ### File Matching
 
-Most managers have a default `fileMatch` array.
-The `fileMatch` array has regular expression strings that match against the repository file list.
+Most Renovate managers have a default `filePatterns` array.
+The `filePatterns` array has a regular expression, or glob pattern, that matches against the repository file list.
```

Comment:
```suggestion
The `filePatterns` array can contain regular expression or glob patterns, that match against the repository file list.
```

Reply:
```suggestion
The `filePatterns` array can hold a regular expression or glob pattern, that match against the repository file list.
```

Grammar fix for your suggestion above, plus change `contain` for `hold`.